docs/methods/activation_functions.md

```diff
@@ -0,0 +1,76 @@
+# Activation Functions 
+
+Sigmoid-It is computationally expensive, causes vanishing gradient problem and not zero-centred. This method is generally used for binary classification problems.
+
+tanh- The Tanh activation function is a hyperbolic tangent sigmoid function that has a range of -1 to 1. It is often used in deep learning models for its ability to model nonlinear boundaries
+
+tan-1h-The inverse of tanh.The ArcTan function is a sigmoid function to model accelerating and decelerating outputs but with useful output ranges.
+
+ReLU-This The ReLU activation function returns 0 if the input value to the function is less than 0 but for any positive input, the output is the same as the input. It is also continuous but non-differentiable at 0 and at values less than 0 because its derivative is 0 for any negative input.
+
+leakyReLU- With Leaky ReLU there is a small negative slope so instead of that firing at all, for large gradients, our neurons do output some value and that makes our layer much more optimized too.
+
+softmax-The softmax is a more generalised form of the sigmoid. It is used in multi-class classification problems. Similar to sigmoid, it produces values in the range of 0–1 therefore it is used as the final layer in classification models.
+
+binaryStep-The Step activation function is used in the perceptron network. This is usually used in single-layer networks to convert to an output that is binary (0 or 1).These are called Binary Step Function.
+
+
+
+## Parameters
+
+| Name         | Definition                                 | Type         |
+|--------------|--------------------------------------------|--------------|
+|   x          |      double value on which the function is applied.                                      | `double` |
+
+
+## Methods
+
+| Name                                   | Definition                                    | Return value  |
+|----------------------------------------|-----------------------------------------------|---------------|
+| y | double value after putting x in the functions gets returned. | `double` |
```

Comment:
- Change `methods` heading to `functions`. 
<details>
<summary> Methods vs functions </summary>

Methods: Function associated with a class. e.g. `fit` function of `LinearRegression` is a method because it is defined inside `LinearRegression` class.
Function: Not associated with a class like your activation functions.

</details>

- Put all function signature and what it does, instead of return type `y`.
  - `sigmoid(x)` | $\frac{1}{1 + e^{-x}}$ | `double` (you can avoid latex part if you want)